udev/vdev_id

```diff
@@ -430,6 +431,12 @@ sas_handler() {
 		d=$(eval echo '$'{$i})
 		SLOT=$(echo "$d" | sed -e 's/^.*://')
 		;;
+	"bay_lun")
```

Comment:
May want a comment here just so the meaning isn't lost to the sands of time:
```
# bay_lun is currently used on multi-actuator HDDs, where one physical drive has two
# LUNs, and thus there isn't a 1:1 LUN -> slot mapping.
```
Feel free to improve upon this language if you want :arrow_up:

Reply:
Good feedback, I added this:
> 		# Like 'bay' but with the LUN number appened. Added for SAS 
> 		# multi-actuator HDDs, where one physical drive has multiple
> 		# LUNs, thus multiple logical drives share the same bay number